colossus/src/main/resources/reference.conf

```diff
@@ -48,7 +48,7 @@ colossus{
       request-buffer-size : 100
       log-errors : true
       request-metrics : true
-      max-request-size : "1 MB"
+      max-request-size : "1000 MB"
```

Comment:
10.MB sounds good to me. Can you update the services docs to explain how to configure a service please. Also your branch needs to be compared to develop branch and not master. All development happens on develop now.